src/compiler/diagnosticMessages.json

```diff
@@ -2824,5 +2824,9 @@
     "Unknown typing option '{0}'.": {
         "category": "Error",
         "code": 17010
+    },
+    "Too many javascript files in the project. Consider add to the `exclude` list in the config file.": {

```

Comment:
Grammar: "Consider adding to...". Or perhaps just reworded as "Use the 'exclude' setting in project configuration to limit included source folders".

Note: Do we predicate this message on the user *not* having listed the files to compile? If we are not scanning directories anyway (i.e. there is a list of files given), then this message would be misleading (as `exclude` can't be specified - at least until the globbing change comes in).

Also, capitalize as "JavaScript", not "javascript", and don't use backticks. This isn't markdown :-p